linux_os/guide/system/accounts/accounts-restrictions/account_unique_id/oval/shared.xml

```diff
@@ -0,0 +1,58 @@
+<def-group>
+  <definition class="compliance" id="account_unique_id" version="1">
+    <metadata>
```

Comment:
please use the [oval_metadata macro](https://complianceascode.readthedocs.io/en/latest/jinja_macros/oval.html?highlight=oval_metadata#template-oval_metadata) here

Reply:
will do